src/App/App.js

```diff
@@ -2,25 +2,52 @@ import './App.css';
 import searchIcon from '../icons/search.png';
 
 // Example imports (for later):
-// import { useState, useEffect } from 'react';
+import { useState, useEffect } from 'react';
 import moviePosters from '../data/movie_posters';
 import movieDetails from '../data/movie_details';
 import MoviesContainer from '../MoviesContainer/MoviesContainer';
 import Header from '../Header/Header';
 
 
 function App() {
+  const [movies, setMovies] = useState([])
 
+  useEffect(() => {
+    setMovies(moviePosters);
+  }, []);
+
+ 
+  const increaseVote = (id) => {
```

Comment:
Great work on this, definitely have to get used to the new react syntax vs vanilla javascript